packages/react-components/react-spinbutton/src/components/SpinButton/useSpinButtonStyles.ts

```diff
@@ -146,50 +208,96 @@ const useRootStyles = makeStyles({
   },
 });
 
+const useInputClassName = makeResetStyles({
+  gridColumnStart: '1',
+  gridColumnEnd: '2',
+  gridRowStart: '1',
+  gridRowEnd: '3',
+  outlineStyle: 'none',
+  border: '0',
+  padding: '0',
+  color: tokens.colorNeutralForeground1,
+  backgroundColor: 'transparent',
```

Comment:
```suggestion
  backgroundColor: tokens.colorTransparentBackground,
```

Reply:
I've added a comment but this is meant to be the literal 'transparent' so the control's background color is set by the root.